client/components/time-since/index.jsx

```diff
@@ -1,40 +1,36 @@
 import moment from 'moment';
```

Comment:
Instead of importing `moment`, it's more correct to use `moment = useLocalizedMoment()` because that will update the formatted strings on locale change.

For the same reason, `getHumanDate` should explicitly get the third `locale` parameter. Also, for full reactivity on locale change, `getHumanDate` should get the `translate` callback as another parameter.

Because `getHumanDate` is used only in this component and nowhere else, it could be convenient to turn it into a `useHumanDate` hook, where the reactivity is easier to implement?